.github/workflows/release.yml

```diff
@@ -212,3 +212,77 @@ jobs:
           body: |
             This is a "canary" release of the most recent commits on our main branch. Canary is **not stable**.
             It is only intended for developers wishing to try out the latest features in Spin, some of which may not be fully implemented.
+
+  create-go-sdk-tag:
+    name: create tag sdk/go/v*
+    runs-on: ubuntu-latest
+    needs: build
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set the tag sdk/go/v*
+        shell: bash
+        run: echo "GO_SDK_TAG=sdk/go/${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Tag sdk/go/v* and push it
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git tag ${{ env.GO_SDK_TAG }}
+          git push origin ${{ env.GO_SDK_TAG }} -f
```

Comment:
I wonder if we want to _not_ use `-f`/`--force`.  I think I'd feel more comfortable erroring out if we're unable to push to origin without forcing.  WDYT?

Reply:
Yeah, sorry, I used `v0.3.0` for testing, so I need to force push to overwrite the tag. I think we don't use the same tag for `sdk/go/v*`, so it's good to remove `-f`. Updated it. Thank you.